corehq/apps/data_pipeline_audit/management/commands/compare_doc_counts.py

```diff
@@ -8,9 +8,9 @@
 
 class Command(BaseCommand):
     help = "Print doc counts in ES & Primary database split by Doc Type."
-    args = '<domain1>..<domainN>'
 
     def add_arguments(self, parser):
+        parser.add_argument('domains', nargs='*')
```

Comment:
if i'm reading [this example](https://docs.djangoproject.com/en/1.11/howto/custom-management-commands/#module-django.core.management) right, this means the list of domains will now be available in `options['domains']`, instead of args, in the handle() method